addons/website_slides/views/slide_slide_views.xml

```diff
@@ -112,6 +132,29 @@
             </field>
         </record>
 
+          <record id="view_slide_question_form" model="ir.ui.view">
+            <field name="name">slide.question.form</field>
+            <field name="model">slide.question</field>
+            <field name="arch" type="xml">
+                <form string="Slide">
+                    <sheet>
+                        <div class="oe_edit_only">
+                            <label for="question" string="Question Name"/>
+                        </div>
+                        <h1>
+                            <field name="question" default_focus="1" placeholder="Name"/>
+                        </h1>
+                        <field name="answer_ids">
+                            <tree editable="bottom" create="true" delete="true">
```

Comment:
Isn't create="true" and delete="true" the default values?
To be checked and removed if it's the case.